### ISSUE TITLE
ChamferLoss seems wrong to use?

### ISSUE THREAD

**Issue by machengcheng2016** [XuyangBai/FoldingNet#1]
`class ChamferLoss(nn.Module):
    def forward(self, preds, gts):
        P = self.batch_pairwise_dist(gts, preds)
`
But when using ChamferLoss, the scripts are:
`output = self.model(pts)
loss = self.model.get_loss(pts, output)`
Which seems just opposite

**Comment by XuyangBai** [comment]
Hi @machengcheng2016 Yes, you are right. This should be a mistake.

**Comment by machengcheng2016** [comment]
Two negatives make a positive, I guess? The codes still run correctly...

**Comment by XuyangBai** [comment]
Yes, because for the Chamfer loss, the roles of two point clouds can be exchanged. So it still works well.
